garage/tf/algos/__init__.py

```diff
@@ -1,8 +1,11 @@
 from garage.tf.algos.batch_polopt import BatchPolopt
 from garage.tf.algos.ddpg import DDPG
 from garage.tf.algos.npo import NPO
+from garage.tf.algos.off_policy_rl_algorithm import OffPolicyRLAlgorithm
 from garage.tf.algos.ppo import PPO
 from garage.tf.algos.trpo import TRPO
 from garage.tf.algos.vpg import VPG
 
-__all__ = ["BatchPolopt", "DDPG", "NPO", "PPO", "TRPO", "VPG"]
+__all__ = [
+    "OffPolicyRLAlgorithm", "BatchPolopt", "DDPG", "NPO", "PPO", "TRPO", "VPG"
```

Comment:
Does this mean that `BatchPolopt` is equivalent to `OnPolicyRLAlgorithm`? Should we then rename it?

Reply:
Yes. `BatchPolopt` refers to all the batch sampling based policy gradient algorithms. But I think it can be used for most on-policy algorithms. (I don't have much knowledge on rl algorithms that are not policy gradient based.)